packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md

```diff
@@ -12,8 +12,10 @@ Create ID Token claims for OpenID Connect or access tokens for OAuth 2.0:
 
     * **Include in token type** &mdash; select **Access Token** (OAuth 2.0) or **ID Token** (OpenID Connect). If you choose **ID Token**, you can also define whether you want the claim included only when requested or always included.
     * **Value type** &mdash; select whether you want to define the claim by a **Groups** filter or by an **Expression** written using the Okta Expression Language.
-    * **Value** &mdash; this option appears if you chose **Expression**. Use Okta Expression Language syntax to generate values derived from attributes in Universal Directory and app profiles. Add the mapping here using the [Okta Expression Language](/docs/reference/okta-expression-language/), for example `appuser.username`.
+    * **Value** &mdash; this option appears if you chose **Expression**. Use Okta Expression Language syntax to generate values derived from attributes in Universal Directory and app profiles. For example `appuser.username`.
       * Check that your expression returns the results expected. You can validate an expression using the **Token Preview** tab.
+      * See [Okta Expression Language](/docs/reference/okta-expression-language) for general information about expressions.
```

Comment:
```suggestion
      * See [Okta Expression Language](/docs/reference/okta-expression-language)
```

Reply:
From the InfoDev style guide, we just have to list the reference (unless there is a specific section in the topic that we are referring to): https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/722929640/InfoDev+Style+Guide#4.-Best-practices-for-InfoDev-styles